code/mod_table/mod_table.cpp

```diff
@@ -418,6 +419,10 @@ void parse_mod_table(const char *filename)
 			stuff_boolean(&Dont_automatically_select_turret_when_targeting_ship);
 		}
 
+		if (optional_string("$Auto select subsystem under reticle when targeting ship:")) {
+			stuff_boolean(&Automatically_select_subsystem_under_reticle_when_targeting_ship);
+		}
```

Comment:
Pedantically speaking, if the subsystem is only selected the second time the ship is targeted, this isn't quite accurate.  You could say "targeting same ship" (and `targeting_same_ship`) as a possible alternative.

Reply:
yeah, I have been struggling to name it something that makes sense and not super long.  I've definitely failed at both.

Reply:
Don't be discouraged; it's a tricky problem to begin with.  These sorts of custom situations are highly specific and often idiosyncratic; witness the "$Don't automatically select a turret when targeting a ship:" right next to it.

And your existing name was 95% of the way there; this is just a suggestion to bring it all the way home.

Reply:
good catch